### PR TITLE
WIP: eh_frame: Full table generation

### PR DESCRIPTION
**Pull request by javierhonduco**
Some outstanding issues:
- PCs (Locs) are wrong
- $rbp offsets are missing
- probably more 😬 
~~- Crash on libc (/lib64/libc.so.6)~~

Signed-off-by: Francisco Javier Honduvilla Coto <javierhonduco@gmail.com>

### CODE REVIEW

**Comment by v-thakkar** [comment]
Merging it and will send the investigation changes on top of it. :laughing:

File: internal/dwarf/frame/table.go [hunk 1]
```diff
@@ -17,22 +17,49 @@ type DWRule struct {
 	Expression []byte
 }
 
-// FrameContext wrapper of FDE context.
-type FrameContext struct {
+// Represents each object code instruction, rather than
```
Comment (kakkoyun) [inline internal/dwarf/frame/table.go:20]: :heart:

File: pkg/stack/unwind/plan_table.go [hunk 2]
```diff
@@ -153,7 +172,7 @@ func (ptb *PlanTableBuilder) readFDEs(path string, start uint64) (frame.FrameDes
 func buildTable(fdes frame.FrameDescriptionEntries, start uint64) PlanTable {
 	table := make(PlanTable, 0, len(fdes))
 	for _, fde := range fdes {
-		table = append(table, buildTableRow(fde, start))
+		table = append(table, buildTableRows(fde, start)[0])
 	}
 	// TODO(kakkayun): Print table and debug.
```
Comment (kakkoyun) [inline pkg/stack/unwind/plan_table.go:177]: I can't believe I have a typo at my handle :see_no_evil:

**Review by kakkoyun — COMMENTED** [review]
LGTM.

We need to think about a strategy for introducing these changes to upstream Delve.

**Pull request merged**